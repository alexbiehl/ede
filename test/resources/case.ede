{ "var": "BiNGo!", "list": ["foo", "bar", "buck"] }
---
    {% case var %}
    {% when "other" %}
      "other"
    {% when "BiNGo!" %}
      "BiNGo!"
    {% when "test" %}
      "test"
    {% endcase %}

{% case 2 %}
{% when 3 %}
3
{% else %}
El fino.
{% endcase %}

  {% for item in list %}
    {% case item.value %}
    {% when "foo" %}
      FOO
    {% when "bar" %}
        BAR
    {% else %}
    BAZ
    {% endcase %}
  {% endfor %}
