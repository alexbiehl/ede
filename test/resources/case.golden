      "BiNGo!"

El fino.

      FOO
        BAR
    BAZ
